apps/pie-storybook/stories/testing/pie-chip.test.stories.ts

```diff
@@ -87,8 +87,14 @@ const chipStoryMeta: ChipStoryMeta = {
 
 export default chipStoryMeta;
 
-const clickAction = action('clicked');
-const closeAction = action('pie-chip-close');
+const clickAction = () => {
+  action('clicked');
+  console.log('pie-chip clicked');
+}
+const closeAction = () => {
+  action('pie-chip-close');
+  console.log('pie-chip-close clicked');
+}
```

Comment:
is there a reason why we are logging two times?
i also think the action isn't needed since it's a testing environment

Reply:
We have tests to ensure the correct event gets emitted depending on which element is clicked. You're right though, i'll remove the `action()` as this isn't being used by the tests